rd_ui/app/scripts/controllers/query_view.js

```diff
@@ -270,9 +278,19 @@
       }
       $scope.selectedTab = hash || DEFAULT_TAB;
     });
+
+    /**
+     * Access groups related
+     */
+    $scope.$watch('query.access_groups', function (newValue, oldValue) {
```

Comment:
Also, you need to update the `Query.newQuery` function to set the `access_groups` value (should probably first copy the value of the data source).